scripts/run-dependencies

```diff
@@ -43,10 +43,51 @@ fi
 
 docker exec daedalus-web-app-db wait-for-db
 
-docker network create daedalus
+NETWORK=daedalus
+VOLUME=daedalus-results
+
+docker network create $NETWORK
 docker network connect daedalus daedalus-web-app-db
 
+docker volume create VOLUME
+
 # Pull and run the R API image
-R_API_IMAGE='mrcide/daedalus.api:latest'
+# TODO: reset this to latest when the rrq branch of the api is merged
+R_API_IMAGE='mrcide/daedalus.api:jidea-74'
 docker pull $R_API_IMAGE # There will be a 'latest' tag: https://github.com/jameel-institute/daedalus.api/pull/2#issuecomment-2252491603
-docker run -d --name daedalus-api --rm --network=daedalus -p 8001:8001 $R_API_IMAGE
+
+QUEUE_ID=daedalus-api-queue-$(uuidgen)
```

Comment:
We could use the same queue id every time, but that can cause problems if the volume doesn't get removed cleanly for some reason, so it's generally safer for the dev/test scripts to generate a new id on every run.

Reply:
So is it the case that we have a single queue at any given time?

Reply:
We should do, yes. But I've had cases of old volumes hanging around and causing problems, and Rich advised that it's standard in the test scripts to just generate a new queue id for yourself every time.